doc/classes/EditorScenePostImport.xml

```diff
@@ -18,7 +18,7 @@
 		    return scene # Remember to return the imported scene
 		func iterate(node):
 		    if node != null:
-		        node.name = "modified_" + node.name
+		        node.name = "modified_" + String(node.name)
```

Comment:
Note that this may not be necessary if https://github.com/godotengine/godot/pull/68747 gets merged.

Reply:
I've removed it for now under the assumption that #68747 is merged